app/controllers/subscribers_auth_token_controller.rb

```diff
@@ -7,7 +7,7 @@ def auth_token
     email = build_email(subscriber, token)
 
     DeliveryRequestWorker
-      .perform_async_in_queue(email.id, queue: :delivery_immediate_high)
+      .perform_async_in_queue(email.id, queue: :delivery_interaction)
```

Comment:
in terms of the queue name, want we going to use this for?
The mention of naming this stuff "transactional emails" seems to fit with the usecase?

Any chance `:delivery_transactional` is more descriptive?

Reply:
Yup - wasn't really sure what to call it. Didn't think it was useful relating it to priority e.g. delivery_immediate_highest, delivery_immediate_utmost_importance

Reply:
lol, sounds like an MS word versioning system `:delivery_immediate_001_FINAL_FINAL_REALLY_FINAL`